Specs/BitmovinAnalyticsCollector/2.0.0-beta1-8-g0388595/BitmovinAnalyticsCollector.podspec.json

```
@@ -0,0 +1,72 @@
+{
+  "name": "BitmovinAnalyticsCollector",
+  "version": "2.0.0-beta1-8-g0388595",
+  "summary": "iOS library that allows you to monitor your iOS video playback with Bitmovin Analytics",
+  "description": "iOS library that allows you to monitor your iOS video playback with Bitmovin Analytics. This SDK can monitor an AVPlayer or a Bitmovin Player",
+  "homepage": "https://github.com/bitmovin/bitmovin-analytics-collector-ios",
+  "license": {
+    "type": "MIT",
+    "file": "LICENSE"
+  },
+  "authors": {
+    "Bitmovin Inc": "admin@bitmovin.com"
+  },
+  "source": {
+    "git": "https://github.com/bitmovin/bitmovin-analytics-collector-ios.git",
+    "tag": "2.0.0-beta1-8-g0388595"
+  },
+  "platforms": {
+    "ios": "12.0",
+    "tvos": "12.0"
+  },
+  "swift_versions": "5.0",
+  "pod_target_xcconfig": {
+    "FRAMEWORK_SEARCH_PATHS": "\"${PODS_ROOT}\"/*",
+    "OTHER_LDFLAGS": "$(inherited) -ObjC",
+    "ENABLE_BITCOD": "YES"
+  },
+  "subspecs": [
+    {
+      "name": "Core",
+      "source_files": "BitmovinAnalyticsCollector/Classes/Collector/**/*.{swift}",
+      "testspecs": [
+        {
+          "name": "CoreTests",
+          "test_type": "unit",
+          "source_files": "BitmovinAnalyticsCollector/Tests/CoreTests/**/*"
+        }
+      ]
+    },
+    {
+      "name": "BitmovinPlayer",
+      "source_files": "BitmovinAnalyticsCollector/Classes/BitmovinPlayerV3/**/*.{swift}",
+      "dependencies": {
+        "BitmovinAnalyticsCollector/Core": [
+
+        ]
```

I'm not 100% familiar with the JSON version of podspecs but AFAIK you don't need the array if you don't specify a version 🤔 (please manually test this)
```suggestion
        "BitmovinAnalyticsCollector/Core"
```

In ruby you would just write:
```
ss.dependency "BitmovinAnalyticsCollector/Core"
```

---

seems like you need to, I can't remember where I looked that up though.